pkg/agent/apiserver/apiserver.go

```diff
@@ -82,6 +83,7 @@ func installHandlers(aq agentquerier.AgentQuerier, npq querier.AgentNetworkPolic
 	s.Handler.NonGoRestfulMux.HandleFunc("/addressgroups", addressgroup.HandleFunc(npq))
 	s.Handler.NonGoRestfulMux.HandleFunc("/ovsflows", ovsflows.HandleFunc(aq))
 	s.Handler.NonGoRestfulMux.HandleFunc("/ovstracing", ovstracing.HandleFunc(aq))
+	s.Handler.NonGoRestfulMux.HandleFunc("/serviceexternalip", serviceexternalip.HandleFunc(seipq))
```

Comment:
you may want to add the `/serviceexternalip` URL to the `antctl` RBAC ClusterRole, just for consistency (and in case someone is using the `antctl` ServiceAccount token to access antrea APIs directly)

Reply:
added.